src/components/timeline/Markers.js

```diff
@@ -210,7 +206,13 @@ class TimelineMarkersImplementation extends React.PureComponent<Props, State> {
     const shouldShowTooltip = !isModifyingSelection && !mouseDownItem;
 
     return (
-      <div className={classNames(className, isSelected ? 'selected' : null)}>
+      <div
+        className={classNames(
```

Comment:
This makes the classes more specific to their actual uses, and removes the risky string concatenation with the thread name.